pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/ufoer/Etteroppgjoer.kt

```diff
@@ -132,15 +132,15 @@ object SoekOmNyInntektsgrense : OutlinePhrase<LangBokmalNynorskEnglish>() {
         title1 {
             text(
                 Bokmal to "Du kan søke om ny inntektsgrense",
-                Nynorsk to "Du kan søkje om ny inntektsgrense",
```

Comment:
Hva er grunnen til at du har fjernet "Du kan" på nynorsk? Bokmål og engelsk har "du kan"/"you can"

Reply:
"Du kan" var fjernet fra/ikke med i NN oversettelsen.